File: BelaVista.Entity/Scheduling.cs
using System;

namespace BelaVista.Entity
{
    public class Scheduling
    {
        public int Id { get; set; }
        public int ScheduleTypeId { get; set; }
        public ScheduleType ScheduleType { get; }
        public int CondominiumId { get; set; }
        public Condominium Condominium { get; }
        public int ScheduleStatusId { get; set; }
        public ScheduleStatus ScheduleStatus { get; set; }
        public DateTime CreateDate { get; set; }
        public DateTime ScheduleDate { get; set; }
        public string Description { get; set; }

        public Scheduling()
        {
            this.CreateDate = DateTime.Now;
        }
    }
}

File: BelaVista-APP/src/app/_models/Scheduling.ts
import { Condominium } from './Condominium';
import { ScheduleType } from './ScheduleType';
import { ScheduleStatus } from './ScheduleStatus';

export interface Scheduling {
    id: number;
    scheduleTypeId: number;
    scheduleStatus: ScheduleStatus;
    schedulyTYpe: ScheduleType;
    condominiumId: number;
    condominium: Condominium;
    scheduleStatusId: number;
    createDate: Date;
    scheduleDate: Date;
    description: string;
}
